internal/backend/backend.go

```diff
@@ -109,7 +109,7 @@ type Backend interface {
 	// DeleteWorkspace cannot prevent deleting a state that is in use. It is
 	// the responsibility of the caller to hold a Lock for the state manager
 	// belonging to this workspace before calling this method.
-	DeleteWorkspace(name string) error
+	DeleteWorkspace(name string, force bool) error
```

Comment:
I dont love adding magic boolean arguments to interfaces, especially when it is only used in one of the backends. I can justify it here because "force deleting" is an [existing concept](https://www.terraform.io/cli/commands/workspace/delete#force) for workspace deleting, which just hasnt been handled by the backends up until now.

However, any suggestions on how to pass this force vs safe deletion boolean to the cloud backend are welcome